watertap/examples/flowsheets/RO_with_energy_recovery/RO_with_energy_recovery.py

```diff
@@ -94,22 +95,21 @@ def build():
     m.fs.product = Product(default={'property_package': m.fs.properties})
     m.fs.disposal = Product(default={'property_package': m.fs.properties})
 
-    # additional variables or expressions
-    product_flow_vol_total = m.fs.product.properties[0].flow_vol
-    m.fs.annual_water_production = Expression(
-        expr=pyunits.convert(product_flow_vol_total, to_units=pyunits.m ** 3 / pyunits.year)
-             * m.fs.costing_param.load_factor)
-    pump_power_total = m.fs.P1.work_mechanical[0] + m.fs.P2.work_mechanical[0]
-    m.fs.specific_energy_consumption = Expression(
-        expr=pyunits.convert(pump_power_total, to_units=pyunits.kW)
-             / pyunits.convert(product_flow_vol_total, to_units=pyunits.m**3 / pyunits.hr))
-
     # costing
-    m.fs.P1.get_costing(module=financials, pump_type="High pressure")
-    m.fs.P2.get_costing(module=financials, pump_type="High pressure")
-    m.fs.RO.get_costing(module=financials)
-    m.fs.PXR.get_costing(module=financials)
-    financials.get_system_costing(m.fs)
+    m.fs.costing.cost_flow(pyunits.convert(m.fs.P1.work_mechanical[0], to_units=pyunits.kW), "electricity")
+    m.fs.costing.cost_flow(pyunits.convert(m.fs.P2.work_mechanical[0], to_units=pyunits.kW), "electricity")
+    m.fs.P1.costing = UnitModelCostingBlock(default={
+            "flowsheet_costing_block":m.fs.costing})
+    m.fs.P2.costing = UnitModelCostingBlock(default={
+            "flowsheet_costing_block":m.fs.costing})
+            
+    m.fs.RO.costing = UnitModelCostingBlock(default={
+            "flowsheet_costing_block":m.fs.costing})
+    m.fs.PXR.costing = UnitModelCostingBlock(default={
+            "flowsheet_costing_block":m.fs.costing})
+    m.fs.costing.cost_process()
+    m.fs.costing.add_LCOW(m.fs.product.properties[0].flow_vol)
+    m.fs.costing.add_specific_energy_consumption(m.fs.product.properties[0].flow_vol)
```

Comment:
There could be cases where we want more than one specific energy consumption (e.g., wrt to product flow and/or wrt feed flow). We could always create an expression ourselves outside of this to calculate an alternative SEC I suppose, but just wanted to point that out. Right now, it looks like calling `add_specific_energy_consumption` more than once would overwrite the previous expression.

Reply:
@adam-a-a At that point however, it is probably easier for the user to create/add the expressions manually. Otherwise, we need to start doing string manipulations, which get messy.

Reply:
That works. I just wanted everyone to be aware of that for SEC (and LCOW); there can be more than one variation of interest in a given analysis.

Reply:
We could provide an optional argument which would allow the user to determine the name of the SEC Expression and LCOW Var/Constraint. That would give flexibility while keeping the simple use case simple (e.g., no weird indexing by flow name or something similar).